src/codeu/chat/client/commandline/Chat.java

```diff
@@ -328,6 +340,166 @@ private ConversationContext find(String title) {
       }
     });
 
+    // I-C-ADD (add conversation interest)
+    //
+    // Add a command that will add a conversation to the current user's
+    // interests when the user enters "i-c-add" while on the user panel
+    // 
+    panel.register("i-c-add", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        for(String token : args){
+          final String title = token;
+          if (title.length() > 0) {
+            final ConversationContext conversation = find(title);
+            if (conversation == null) {
+              System.out.format("ERROR: No conversation with name '%s'\n", title);
+            } else {
```

Comment:
An `else if` block can be used here to condense the code and avoid creating another nested `if` block. This change should be applied to the applicable `if` blocks in the rest of the methods (like i-c-remove and i-u-add).